include/torch-mlir/Conversion/Passes.td

```diff
@@ -114,6 +114,35 @@ def ConvertTorchToTosa : Pass<"convert-torch-to-tosa", "func::FuncOp"> {
   let constructor = "mlir::torch::createConvertTorchToTosaPass()";
 }
 
+def ConvertTorchToTosaCustom : Pass<"convert-torch-to-tosa-custom", "func::FuncOp"> {
+        let summary = "Convert Torch ops to TOSA custom ops";
+  let description = [{
+    The purpose to use tosa::custom is handle complex ops when we donnot
+    want to decompose them into simple ops.
+    The aten op name will used to construct a StringAttr as the identifier attribute for tosa::CustomOp.
+    So in the output the users know where is the tosa::CustomOp is coming from by the StringAttr id.
+    Each input arg from Aten Dialect has to be converted to a tensor of number values as the
+    operand of tosa::CustomOp op. After convert, use ValueRange/SmallVector to include
+    all operands as the final input operands for tosa::CustomOp.
+    The contract to follow:
+        AnyTorchTensorType 	->	TensorType<?xAnyType>  of tosa::ConstOp
```

Comment:
nit: I would replace `?` with `AnySize`, since `?` means the type has a single dynamic dimension

Reply:
Done